src/script/components/calling/GroupVideoGridTile.tsx

```diff
@@ -149,6 +149,7 @@ const GroupVideoGridTile: React.FC<GroupVideoGridTileProps> = ({
           <Video
             autoPlay
             playsInline
+            muted // This is needed to keep playing the video when detached to a new window
```

Comment:
could you link some external doc that reference this and the reason why?

Reply:
sure, will do